library_schemas/score/prerelease/HED_score_1.0.0.mediawiki

```diff
@@ -56,35 +56,35 @@ This project was supported by the by the National Institute of Mental Health of
         ** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
     * Cognitive-task
         ** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-    * Other-modulator-or-procedure
+    * Other-modulator-or-procedure {requireChild,extensionAllowed}
```

Comment:
You should take out the extensionAllowed.  The # and extensionAllowed are conflicts. You cannot add another node below Other-modulator-or-procedure.  The # says that you can add a value.

Reply:
Thanks for pointing that out!
So either a childe node that takes value like this:

> <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>

or the extensionAllowed schema attribute?

Reply:
This is still a **draft** pull request correct? Let me know when you want me to comment.